include/deal.II/distributed/tria_base.h

```diff
@@ -151,6 +151,14 @@ namespace parallel
     const std::set<types::subdomain_id> &
     level_ghost_owners () const;
 
+    /**
+     * Return a map that, for each vertex, lists all the processors whose
+     * subdomains are adjacent to that vertex. Used by
+     * DoFHandler::Policy::ParallelDistributed.
+     */
+    virtual std::map<unsigned int, std::set<dealii::types::subdomain_id> >
+    compute_vertices_with_ghost_neighbors () const;
```

Comment:
it is in this change in tria_base.cc and I need it because it is currently only implemented in p::d::Tria and I didn't want to restrict this function to that.